src/CommunityToolkit.Maui.Markup/ViewInAbsoluteLayoutExtensions.cs

```diff
@@ -0,0 +1,138 @@
+	/// <param name="view"></param>
+	/// <param name="flag"></param>
+	/// <returns></returns>
+	public static TView LayoutFlags<TView>(this TView view, AbsoluteLayoutFlags flag) where TView : View
+	{
+		AbsoluteLayout.SetLayoutFlags(view, flag);
+		return view;
+	}
+
+	/// <summary>
+	/// Set LayoutBounds
+	/// </summary>
+	/// <typeparam name="TView"></typeparam>
+	/// <param name="view"></param>
+	/// <param name="x"></param>
+	/// <param name="y"></param>
+	/// <returns></returns>
+	public static TView LayoutBounds<TView>(this TView view, double x, double y) where TView : View
+	{
+		Rect rect = new(x, y, AbsoluteLayout.AutoSize, AbsoluteLayout.AutoSize);
+		AbsoluteLayout.SetLayoutBounds(view, rect);
+		return view;
+	}
+
+
+	/// <summary>
+	/// Set LayoutBounds
+	/// </summary>
+	/// <typeparam name="TView"></typeparam>
+	/// <param name="view"></param>
+	/// <param name="point"></param>
+	/// <returns></returns>
+	public static TView LayoutBounds<TView>(this TView view, Point point) where TView : View
+	{
+		Rect rect = new(point.X, point.Y, AbsoluteLayout.AutoSize, AbsoluteLayout.AutoSize);
+		AbsoluteLayout.SetLayoutBounds(view, rect);
+		return view;
+	}
+
+	/// <summary> 
+	/// Set LayoutBounds 
+	/// </summary> 
+	/// <typeparam name="TView"></typeparam> 
+	/// <param name="view"></param> 
+	/// <param name="bounds"></param> 
+	/// <returns></returns> 
+	public static TView LayoutBounds<TView>(this TView view, Rect bounds) where TView : View
+	{
+		AbsoluteLayout.SetLayoutBounds(view, bounds);
+		return view;
+	}
+
+	/// <summary> 
+	/// Set LayoutBounds
+	/// </summary> 
+	/// <typeparam name="TView"></typeparam> 
+	/// <param name="view"></param> 
+	/// <param name="point"></param> 
+	/// <param name="size"></param> 
+	/// <returns></returns> 
+	public static TView LayoutBounds<TView>(this TView view, Point point, Size size) where TView : View
+	{
+		Rect rect = new(point, size);
+		AbsoluteLayout.SetLayoutBounds(view, rect);
+		return view;
+	}
+
+	/// <summary>
+	/// Set LayoutBounds
+	/// </summary>
+	/// <typeparam name="TView"></typeparam>
+	/// <param name="view"></param>
+	/// <param name="x"></param>
+	/// <param name="y"></param>
+	/// <param name="size"></param>
+	/// <returns></returns>
+	public static TView LayoutBounds<TView>(this TView view, double x, double y, Size size) where TView : View
+	{
+		Rect rect = new(x, y, size.Width, size.Height);
+		AbsoluteLayout.SetLayoutBounds(view, rect);
+		return view;
+	}
+
+	/// <summary>
+	/// Set LayoutBounds
+	/// </summary>
+	/// <typeparam name="TView"></typeparam>
+	/// <param name="view"></param>
+	/// <param name="point"></param>
+	/// <param name="width"></param>
+	/// <param name="height"></param>
+	/// <returns></returns>
+	public static TView LayoutBounds<TView>(this TView view, Point point, double width, double height) where TView : View
+	{
+		Rect rect = new(point.X, point.Y, width, height);
+		AbsoluteLayout.SetLayoutBounds(view, rect);
+		return view;
+	}
```

Comment:
Do we need these overloads? I like having the signature follow a similar pattern such as all complex types and all primitive types. Of course I haven't compared this to the rest of the framework so I may be alone in my opinion here

Reply:
Makes sense, we can have both overloads, but just one implementation, and reusing the implementation on the other

Reply:
> Makes sense, we can have both overloads, but just one implementation, and reusing the implementation on the other

What do you mean by just one implementation?

Reply:
> @danielftz something like this
> 
> ```cs
> public static TView LayoutBounds<TView>(this TView view, double x, double y, Size size) where TView : View
> {
>     Rect rect = new(x, y, size.Width, size.Height);
>     AbsoluteLayout.SetLayoutBounds(view, rect);
>     return view;
> }
> 
> public static TView LayoutBounds<TView>(this TView view, Point point, double width, double height) where TView : View
> {
>     return view.LayoutBounds(point.X, point.Y, new Size(width, height));
> }
> ```

All the overloads can use just one implementation then

Reply:
@danielftz if possible yes, that means less code to maintain :D

Reply:
What about thoughts on just matching what the .NET MAUI methods provide? https://docs.microsoft.com/en-gb/dotnet/maui/user-interface/layouts/absolutelayout#position-and-size-children Or do we want all of these possibilities?

Reply:
> What about thoughts on just matching what the .NET MAUI methods provide? https://docs.microsoft.com/en-gb/dotnet/maui/user-interface/layouts/absolutelayout#position-and-size-children Or do we want all of these possibilities?
Personally I don't see any harm with having more options, however I'm also good with reducing them. As long as mods can agree on it.

Reply:
I agree that these options are good. Plus, @danielftz has already provided Unit Tests, and all of the additional methods leverage the two base methods of `LayoutFlags` and `LayoutBounds`.

I can see these overloaded methods making their way into `Microsoft.Maui.Controls` once we prove their value in the Toolkit 💯